src/Movie/Movie.js

```diff
@@ -0,0 +1,38 @@
+import './Movie.css'
+
+const Movie = ({ movieDetails }) => {
+    const {
+        title, posterPath, backdropPath, releaseDate, overview,
+        genres, budget, revenue, runtime, tagline, averageRating
+    } = movieDetails;
```

Comment:
ok idk what is going on here but it looks cool! Looks like you're breaking up the movieDetails into those constants? Do the curlies make it know that those are key values? Would love this explained like I'm 5 lol